go/client/client.go

```diff
@@ -244,7 +244,7 @@ func (c *Client) WatchBlocks(ctx context.Context, runtimeID signature.PublicKey)
 // Pass RoundLatest to get the latest block.
 func (c *Client) GetBlock(ctx context.Context, runtimeID signature.PublicKey, round uint64) (*block.Block, error) {
 	if round == RoundLatest {
-		return c.common.roothash.GetLatestBlock(ctx, runtimeID)
+		return c.common.roothash.GetLatestBlock(ctx, runtimeID, 0)
```

Comment:
Hmm, random thought.  There's a few places where we do this.  I wonder if people will confuse RoundLatest with 0.  Depending on the API, certain places expect one or the other...

Reply:
So one is latest roothash round the other is latest consensus height.

It can be confusing yeah, not sure how to improve on that?

Reply:
Fair enough.